daemons/dss-sync-sfn/app.py

```diff
@@ -49,7 +49,9 @@ def launch_from_s3_event(event, context):
                 if exists(dest_replica, obj.key):
                     # Logging error here causes daemons/invoke_lambda.sh to report failure, for some reason
                     # - Brian Hannafious, 2019-01-31
-                    logger.info("Key %s already exists in %s, skipping sync", obj.key, dest_replica)
```

Comment:
Since we only have two replicas, we could infer which function was hit from `dst_replica`?

Reply:
Yes, it's implied, but I'd rather it be explicit and know since bugs happen which things hit the wrong target.